src/ultk/effcomm/rate_distortion.py

```diff
@@ -0,0 +1,191 @@
+"""Helper functions for Rate-Distortion based (including Information Bottleneck) efficient communication analyses."""
+
+import numpy as np
+from ultk.language.language import Language
+from ultk.effcomm.agent import LiteralSpeaker, Listener
+from rdot.ba import IBOptimizer, IBResult
+from rdot.probability import joint, bayes
+from rdot.information import information_cond, MI
+
+##############################################################################
+# Measuring Languages
+##############################################################################
+
+
+def language_to_ib_coordinate(
+    language: Language,
+    prior: np.ndarray,
+    meaning_dists: np.ndarray,
+) -> tuple[float]:
+    """Compute the complexity, informativity, and communicative cost under the IB framework (Zaslavsky et al., 2018, i.a.).
+
+    Args:
+        language: the Language to measure. This function will infer an IB encoder, which will be represented as a 2D array of shape `(|words|, |meanings|)`
+
+        meaning_dists: array of shape `(|meanings|, |meanings|)` representing the distribution over world states given meanings.
+
+        prior: array of shape `|meanings|` representing the communicative need distribution
+
+    Returns:
+        a tuple of floats `(complexity, accuracy, distortion)`, s.t.
+
+            `complexity`: the complexity of the lexicon I[M:W], in bits
+
+            `accuracy`: the accuracy of the lexicon I[W:U], in bits
+
+            `distortion`: the distortion E[DKL[ M || M_hat ]] = I[M:U] - I[W:U], in bits
+    """
+    args = [language, prior, meaning_dists]
+    return ib_encoder_to_point(
+        *args[1:], language_to_ib_encoder_decoder(*args).values()
```

Comment:
Does the `language_to...` not also need a `*` in front of it?

Reply:
Actually, I feel like this might be easier to read if you just get the raw dict from `language_to_ib....` in the body, and then send in in `["encoder"]` and `["decoder"]` here